wordpress-mobile/WordPress-iOS
# Issue/3271 add support for animated gifs on post list

Fixes #3271 #2752 

This PR add supported for animated GIF on post list by using the FLAnimatedImage library from Flipboard

To test:
 - Open the app
 - Create a new post
 - Add image to it that is an animated GIF, and set it has featured image, or only add one image to the post
 - Post it
 - Check on the post list that  the animated GIF shows correctly and plays correctly.
 - Try add multiple ones and see if they work correctly and there is no performance or memory issues.

Needs review: @aerych
